src/main.cc

```diff
@@ -584,7 +585,13 @@ std::vector<Option> options_
         "Currently supported settings are 'numa-node', 'cpu-cores'. Note that "
         "'numa-node' setting will affect pinned memory pool behavior, see "
         "--pinned-memory-pool for more detail."
-  }
+  },
+      {OPTION_DEFAULT_MAX_BATCH_SIZE, "default-max-batch-size", Option::ArgInt,
```

Comment:
Do we need to support this mode along with `--backend-config=<default-max-batch-size>`?

Reply:
Yes we should be adding it as a backend-config. I would rather not pollute the argument names for backend specific parameters. This is why we use backend-config

Reply:
See [this comment](https://github.com/triton-inference-server/server/pull/4092#discussion_r840810549).